### PR TITLE
feat: tweak `CompilerPerf`

### PR DESCRIPTION
**Pull request by magnus-madsen**

### CODE REVIEW

File: main/src/ca/uwaterloo/flix/tools/CompilerPerf.scala [hunk 1]
```diff
@@ -267,7 +268,7 @@ object CompilerPerf {
       ("lines" -> lines) ~
       ("results" -> baseline.phases.zip(baselineWithPar.phases).map {
         case ((phase, times1), (_, times2)) =>
-          ("phase" -> phase) ~ ("speedup" -> combine(times1) / combine(times2))
+          ("phase" -> phase) ~ ("speedup" -> combine(times1.zip(times2).map(p => p._1 / p._2)))
```
Comment (magnus-madsen) [inline main/src/ca/uwaterloo/flix/tools/CompilerPerf.scala:271]: Imagine if I could do `map(/)` or `map(Div.div)`-- Well in Flix I will soon be able!

Comment (magnus-madsen) [reply main/src/ca/uwaterloo/flix/tools/CompilerPerf.scala:271]: Hell, even today I can do `map(match (x, y) -> x / y`.

File: main/src/ca/uwaterloo/flix/tools/CompilerPerf.scala [hunk 2]
```diff
@@ -211,10 +211,11 @@ object CompilerPerf {
    * The combine function uses to merge data from different runs.
    */
   def combine[T](xs: Seq[T])(implicit numeric: Numeric[T]): Double =
-    if (N <= 5)
+    if (N <= 4)
       numeric.toDouble(xs.last)
-    else
+    else {
       StatUtils.median(xs)
+    }
```
Comment (mlutze) [inline main/src/ca/uwaterloo/flix/tools/CompilerPerf.scala:218]: What a weird style guide you follow. "Braces are mandatory only for the else clause"

File: main/src/ca/uwaterloo/flix/util/StatUtils.scala [hunk 3]
```diff
@@ -22,7 +22,7 @@ object StatUtils {
   /**
     * Returns the average of `xs`.
     */
-  def avg[T](xs: Seq[T])(implicit numeric: Numeric[T]): Double = {
+  def average[T](xs: Seq[T])(implicit numeric: Numeric[T]): Double = {
```
Comment (mlutze) [inline main/src/ca/uwaterloo/flix/util/StatUtils.scala:25]: "mean" would be more precise

**Review by mlutze — APPROVED** [review]

File: main/src/ca/uwaterloo/flix/tools/CompilerPerf.scala (hunk 2 above)
Comment (magnus-madsen) [reply main/src/ca/uwaterloo/flix/tools/CompilerPerf.scala:218]: Fixed! I was messing around 👍

**Pull request merged**